python/ros_interface.py

```diff
@@ -0,0 +1,110 @@
+import rospy
+from sailbot_msg.msg import Sensors, windSensor, GPS
+
+# Constants
+CHECK_PERIOD_SECONDS = 0.1  # How often fields are updated
+KNOTS_TO_KMPH = 1.852
+ERROR = 0.2  # If signals need different margin of errors, modify as required
+
+"""
+Conversion Notes:
+- wind_speedKmph, wind_direction, wind_headingDegrees, gps_speedKmph: int32 -> float64 (no loss of precision)
+- wind_speedKmph, gps_speedKmph: knots -> km/h
+- wind_direction: no conversion required (0 is bow, 90 is starboard/right, etc - cw - degrees)
```

Comment:
I think this actually isn't right:

From utilities.py:
```
# Constants for boat frame angles
BOAT_RIGHT = 0
BOAT_FORWARD = 90
BOAT_LEFT = 180
BOAT_BACKWARD = 270
```

0 is right, not 90 in our code. I also think that our code treats it as ccw. Need to change this comment and add another conversion to the code.